packages/medusa/src/loaders/models.ts

```diff
@@ -5,30 +5,76 @@ import { ClassConstructor, MedusaContainer } from "../types/global"
 import { EntitySchema } from "typeorm"
 import { asClass, asValue } from "awilix"
 
+function formatModelName(modelName) {
+  // Remove "Model" from the end of the string
+  let formattedName = modelName.replace('Model', '');
+
+  // Capitalize the first letter of the string
+  formattedName = formattedName.charAt(0).toUpperCase() + formattedName.slice(1);
+
+  return formattedName;
+}
+
 /**
  * Registers all models in the model directory
  */
 export default (
-  { container, isTest }: { container: MedusaContainer; isTest?: boolean },
+  { container, isTest, rootDirectory }: { container: MedusaContainer; isTest?: boolean, rootDirectory?: string },
```

Comment:
todo: I am not sure this is the right place to do it, here the plugins are not taken into account. Could we move that to a separate tasks that we run independently in the index loader? Maybe a new function in the plugins loader that we call from the index directly

Reply:
as discussed, we're running this in both places to keep things sane for the PR